seekarc/res/drawable/seek_arc_control_selector.xml

```diff
@@ -23,9 +23,9 @@
 -->
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <item android:drawable="@drawable/scrubber_control_disabled_holo" android:state_enabled="false"/>
-    <item android:drawable="@drawable/scrubber_control_pressed_holo" android:state_pressed="true"/>
-    <item android:drawable="@drawable/scrubber_control_focused_holo" android:state_selected="true"/>
-    <item android:drawable="@drawable/scrubber_control_normal_holo"/>
+    <item android:drawable="@drawable/scrubber_control_disabled_holo" android:state_enabled="false" />
+    <item android:drawable="@drawable/scrubber_control_pressed_holo" android:state_pressed="true" />
+    <item android:drawable="@drawable/scrubber_control_focused_holo" android:state_selected="true" />
+    <item android:drawable="@drawable/scrubber_control_normal_holo" />
```

Comment:
Same comment about not reformatting code in this module. It's ~blatantly stolen~ borrowed (and modified) from [neild001/SeekArc](https://github.com/neild001/SeekArc), so I'd prefer to keep the code in-sync as much as possible.

The changes to the Gradle file are fine, though, as I already had to modify it in the past.